airbyte-integrations/connectors/source-gitlab/source_gitlab/streams.py

```diff
@@ -0,0 +1,334 @@
+                    if not self.project_ids or self.project_ids and pid in self.project_ids:
+                        yield {"id": pid.replace("/", "%2F")}
+        else:
+            for pid in self.project_ids:
+                yield {"id": pid.replace("/", "%2F")}
+
+
+class Milestones(GitlabChildStream):
+    pass
+
+
+class Members(GitlabChildStream):
+    flatten_parent_id = True
+
+
+class Labels(GitlabChildStream):
+    flatten_parent_id = True
+
+
+class Branches(GitlabChildStream):
+    primary_key = "name"
+    flatten_id_keys = ["commit"]
+    flatten_parent_id = True
+
+
+class Commits(IncrementalGitlabChildStream):
+    cursor_field = "created_at"
+    filter_field = "since"
+    flatten_parent_id = True
+    stream_base_params = {"with_stats": True}
+
+
+class Issues(IncrementalGitlabChildStream):
+    stream_base_params = {"scope": "all"}
+    flatten_id_keys = ["author", "assignee", "closed_by", "milestone"]
+    flatten_list_keys = ["assignees"]
+
+
+class MergeRequests(IncrementalGitlabChildStream):
+    stream_base_params = {"scope": "all"}
+    flatten_id_keys = ["author", "assignee", "closed_by", "milestone", "merged_by"]
+    flatten_list_keys = ["assignees"]
+
+
+class MergeRequestCommits(GitlabChildStream):
+    path_list = ["project_id", "iid"]
+
+    path_template = "projects/{project_id}/merge_requests/{iid}"
+
+    def transform(self, record, stream_slice: Mapping[str, Any] = None, **kwargs):
+        record["project_id"] = stream_slice["project_id"]
+        record["merge_request_iid"] = stream_slice["iid"]
+
+        return record
+
+
+class Releases(GitlabChildStream):
+    primary_key = "name"
+    flatten_id_keys = ["author", "commit"]
+    flatten_list_keys = ["milestones"]
+
+    def transform(self, record, stream_slice: Mapping[str, Any] = None, **kwargs):
+        super().transform(record, stream_slice, **kwargs)
+        record["project_id"] = stream_slice["id"]
+
+        return record
+
+
+class Tags(GitlabChildStream):
+    primary_key = "name"
+    flatten_id_keys = ["commit"]
+
+    def transform(self, record, stream_slice: Mapping[str, Any] = None, **kwargs):
+        super().transform(record, stream_slice, **kwargs)
+        record["project_id"] = stream_slice["id"]
+
+        return record
+
+
+class Pipelines(IncrementalGitlabChildStream):
+    pass
+
+
+class PipelinesExtended(GitlabChildStream):
+    path_list = ["project_id", "id"]
+    path_template = "projects/{project_id}/pipelines/{id}"
+
+
+class Jobs(GitlabChildStream):
+    flatten_id_keys = ["user", "pipeline", "runner", "commit"]
+    path_list = ["project_id", "id"]
+    path_template = "projects/{project_id}/pipelines/{id}/jobs"
+
+    def transform(self, record, stream_slice: Mapping[str, Any] = None, **kwargs):
+        super().transform(record, stream_slice, **kwargs)
+        record["project_id"] = stream_slice["project_id"]
+        return record
+
+
+class Users(GitlabChildStream):
+    pass
+
+
+# TODO: No permissions to check
```

Comment:
can we fix this comment and others, it should say that we need to upgrade the plan for these feature (epics) to be available